06-func.md

```diff
@@ -372,9 +369,8 @@ difference in standard deviations before and after: -3.5527136788e-15
 Again,
 the difference is very small.
 It's still possible that our function is wrong,
-but it seems unlikely enough that we should probably get back to doing our analysis.
-We have one more task first, though:
-we should write some [documentation](reference.html#documentation) for our function
+but it seems unlikely therefore we should probably get back to doing our analysis.
```

Comment:
Here also, we could use "so" but not "therefore". I kind of like the original in this case though...